### DISCUSSION TITLE
How to use with SSH

### DISCUSSION THREAD

**Discussion by MordechaiHadad** [Vinzent03/obsidian-git#571] [Q&A] [answered]
How can I give this plugin my SSH password so that it uses it whenever I push? kind of like vscode

**Comment by Vinzent03** [comment, marked as answer]
There is no way to configure this in the plugin settings. This has to be configured for the system installed git. So it has to be added to the ssh-agent. [Doc](https://publish.obsidian.md/git-doc/Authentication) might help as well.

**Reply by MordechaiHadad** [reply]
oh... I thought it was something settings wise, since a friend of mine uses the same plugin but got prompted to enter his password.
weird.
